packages/runtime/src/templates/handlerUtils.ts

```diff
@@ -1,11 +1,10 @@
 import fs, { createWriteStream, existsSync } from 'fs'
-import http from 'http'
-import https from 'https'
 import { tmpdir } from 'os'
 import path from 'path'
 import { pipeline } from 'stream'
 import { promisify } from 'util'
 
+import { http, https } from 'follow-redirects'
```

Comment:
I checked and the license for this package is MIT, so it's fine to add it. There's a `maxRedirects` config for this package that defaults to 21.

How many redirects currently occur? I would assume 1? So maybe lower this default limit to 1?

Instead of potentially adding this package, can there be a check if the HTTP status code is a redirect and react accordingly?

Reply:
> How many redirects currently occur? I would assume 1? So maybe lower this default limit to 1?

In this case it was 1 so I'd be ok to lower the default of `maxRedirects` to `1`.

> Instead of potentially adding this package, can there be a check if the HTTP status code is a redirect and react accordingly?

We could but I think that we shouldn't in this case. This is a very popular library with a good amount of [test coverage](https://github.com/follow-redirects/follow-redirects/blob/96552371eeec58567850d0288ba793274d1ca50b/test/test.js) and I consider using this less risky compared to doing something ourselves here.

Reply:
Sounds good @ericapisani. I don't mind bringing in packages, just wanted to see if it was worth it is all.